src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -248,16 +248,17 @@ const SulPeopleTableView = ({items, hasHeading}: Props) => {
                 <Td className="block w-auto px-0 py-16 text-center md:border-b md:border-black-40 md:px-9 md:py-16 md:text-left lg:table-cell lg:w-1/5 lg:pr-72">
                   {item.email && (
                     <>
-                      <EnvelopeIcon
-                        title="Email"
-                        width={20}
-                        className="mr-6 inline-block text-digital-blue"
-                      />
-
-                      <EmailLink
-                        email={item.email}
-                        className="break-words text-16 font-normal leading-[23px] text-digital-blue underline transition-colors hover:bg-black-10 hover:text-brick-dark focus:bg-none focus:text-cardinal-red active:text-cardinal-red"
-                      />
+                      <Link
```

Comment:
Set the prop `prefetch={false}` when linking to outside places like mailto or other sites

Reply:
Good to know.  I looked it up and now know why. :)